lib/structures/Invite.js

```diff
@@ -3,28 +3,32 @@
 const Base = require("./Base");
 
 /**
-* Represents an invite. The nullable properties will be null if the bot user does not have manage channel or manage server permissions for the invite's channel/server.
-* @prop {String} code The invite code
-* @prop {Object} channel Info on the invite channel
-* @prop {String} channel.id The ID of the invite's channel
-* @prop {String} channel.name The name of the invite's channel
-* @prop {Object?} guild Info on the invite guild
-* @prop {String} guild.id The ID of the invite's guild
-* @prop {String} guild.name The name of the invite's guild
-* @prop {String?} guild.splash The hash of the invite splash screen
-* @prop {String?} guild.icon The hash of the guild icon
-* @prop {Number?} guild.textChannelCount The number of text channels in the guild
-* @prop {Number?} guild.voiceChannelCount The number of voice channels in the guild
-* @prop {User?} inviter The invite creator
-* @prop {Number?} uses The number of invite uses
-* @prop {Number?} maxUses The max number of invite uses
-* @prop {Number?} maxAge How long the invite lasts in seconds
-* @prop {Boolean?} temporary Whether the invite is temporary or not
-* @prop {Number?} createdAt Timestamp of invite creation
-* @prop {Boolean?} revoked Whether the invite was revoked or not
-* @prop {Number?} presenceCount The **approximate** presence count for the guild
-* @prop {Number?} memberCount The **approximate** member count for the guild
-*/
+ * Represents an invite. Some properties are only available when fetching invites from channels, which requires the Manage Channel permission.
+ * @prop {String} code The invite code
+ * @prop {Object} channel Info on the invite channel
+ * @prop {String} channel.id The ID of the invite's channel
+ * @prop {String} channel.name The name of the invite's channel
+ * @prop {Number} channel.type The type of the invite's channel
+ * @prop {Object?} guild Info on the invite guild
+ * @prop {String} guild.id The ID of the invite's guild
+ * @prop {String} guild.name The name of the invite's guild
+ * @prop {String?} guild.splash The hash of the invite splash screen
+ * @prop {String?} guild.banner The hash of the guild banner
+ * @prop {String?} guild.description The description of the invite's guild
+ * @prop {String?} guild.icon The hash of the guild icon
+ * @prop {String[]} guild.features An array of the invite's guild's features
+ * @prop {Number?} guild.verificationLevel The verification level of the invite's guild
+ * @prop {String?} guild.vanityUrlCode The vanity url of the invite's guild
+ * @prop {User?} inviter The invite creator
+ * @prop {Number?} uses The number of invite uses
+ * @prop {Number?} maxUses The max number of invite uses
+ * @prop {Number?} maxAge How long the invite lasts in seconds
+ * @prop {Boolean?} temporary Whether the invite is temporary or not
+ * @prop {Number?} createdAt Timestamp of invite creation
+ * @prop {Boolean?} revoked Whether the invite was revoked or not
```

Comment:
This was removed in typings, should it not also be removed here?

Reply:
I would also check if there is code below in the constructor for this as well and remove that.

Reply:
revoked was removed from the typings, but the rest were not